src/redux/pattern/pattern.reducer.js

```diff
@@ -0,0 +1,23 @@
+import { SET_PATTERNS, SET_PATTERN_LOADING } from './pattern.types';
+
+export const initialState = {
+  list: []
+};
+
+const patternReducer = (state = initialState, action = {}) => {
+  if (action.type === SET_PATTERNS) {
```

Comment:
Use `switch/case` construction here, we use it in every `reducer` file, also if this reducer becomes bigger `if/else` will not be a good choice

Reply:
done